ci/travis/astyle.sh

```diff
@@ -19,6 +21,13 @@ then
 	fi
 fi
 
+is_valid_file(){
```

Comment:
maybe let's name this function `is_ignored`

Reply:
This won't make sense when interpreting line:
`if is_source_file "$file" && is_ignored "$file"`

Natural language: if the current file is a source file and is ignored then run astyle

Reply:
it would be in the sense of `if is_source_file "$file" && ! is_ignored "$file"`
but no strong opinion; we can leave it like this for now

it could also be implemented something like where everything is moved under `is_valid_file` and `is_valid_file` could check if it is ignored, a source file, etc; it would also be extendable;

Reply:
Yep, that's how I initially did it: check both if ignored and if it s source file within same function i.e. `is_source_file`. I can implement it like that.

Reply:
let's leave it like this for now